helm/openwhisk/values.yaml

```diff
@@ -136,7 +136,13 @@ k8s:
 # Images used to run auxillary tasks/jobs
 utility:
   imageName: "openwhisk/ow-utils"
-  imageTag: "00fad95"
+  imageTag: "3a6f6a7"
+  imagePullPolicy: "IfNotPresent"
+
+# Images used to run auxillary tasks/jobs
+wskadmin:
+  imageName: "openwhisk/ow-utils"
+  imageTag: "3a6f6a7"
   imagePullPolicy: "IfNotPresent"
```

Comment:
I think you can drop the addition of wskadmin stanze to values.yaml since the pod spec is using `Values.utiliity.imageName` etc.